docs/technical_debt.md

```diff
@@ -14,8 +11,9 @@ branching nests of conditionals. We're trying a few techniques on this
 project to forestall that that could be summarized with the following principles:
 
 1. Long model classes are bad
-2. NullObject is better than nil
-3. View logic should not be in the templates
+2. Composition/delegation is better than inheritance
```

Comment:
we do have some inheritance still, eg: `BaseRules`. I think inheritance is fine as long as it is "shallow and wide"